ansible/roles/ansible-tvault-contego-extension/tasks/main.yml

```diff
@@ -5,7 +5,17 @@
        - contego
        - contego-extension
        - all-install
-
+       
+- debug:
+    msg:
+        - "You defined 'UPDATE_NOVA_SUDOERS_FILE' as  'notproceed' for updating nova sudoers file,"
+        - "Inside the environments/group_vars/all/vars.yml "
+        - "Exiting the Contego installation"
+        - "Please define varibale  'UPDATE_NOVA_SUDOERS_FILE' as  'proceed' inside the environments/group_vars/all/vars.yml"
+
```

Comment:
Don't we need to show this message only on the condition of UPDATE_NOVA_SUDOERS_FILE =='notproceed' ?

Reply:
Requested changes done